### PR TITLE
Openai embedding fix to support jina-embeddings-v2

### PR DESCRIPTION
**Pull request by wizd**
## Checklist:

- [x] I have read the [Contributing guidelines](https://github.com/oobabooga/text-generation-webui/wiki/Contributing-guidelines).

tested with: https://huggingface.co/jinaai/jina-embeddings-v2-base-en

### CODE REVIEW

**Comment by wizd** [comment]
tested embedding model:
```
jinaai/jina-embeddings-v2-base-en
BAAI/bge-large-zh-v1.5
```

File: extensions/openai/completions.py [hunk 1]
```diff
@@ -235,7 +235,7 @@ def chat_completions_common(body: dict, is_legacy: bool = False, stream=False) -
 
     max_tokens = generate_params['max_new_tokens']
     if max_tokens in [None, 0]:
-        generate_params['max_new_tokens'] = 4096
+        generate_params['max_new_tokens'] = 200
```
Comment (oobabooga) [inline extensions/openai/completions.py:238]: Setting this high isn't necessary, as the auto_max_new_tokens fills the context. The 200 reference value is used when the context is fully used to remove old messages.

**Comment by oobabooga** [comment]
Looks good, thanks

**Comment by yhyu13** [comment]
@wizd Does Jina embedding runs on cuda devices? Other setense transformer embedding always lies on CPU even if specifiying cuda

**Pull request merged**